streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -117,7 +118,7 @@
      * });
      * }</pre>
      * <p>
-     * The provided {@link KeyValueMapper} must return a {@link KeyValue} type and the return value must not be {@code null}.
+     * The provided {@link KeyValueMapper} must return a {@link KeyValue} type and must not return {@code null}.
```

Comment:
Hmm good point. Maybe we should consider ignore nulls in maps if users found it not very friendly. Anyways, just a reminder for future changes.